index.ipynb

```diff
@@ -82,7 +82,21 @@
     "\n",
     "Alternatively, you can also download individual notebook files (with the extension `.ipynb`) and open them in IPython.\n",
     "Note that some exercises make use of additional files (audio files etc.) which you'll then also have to download manually.\n",
-    "\n"
+    "\n",
+    "### Anconda Usage\n",
+    "\n",
+    "If we want to utilize the [Anaconda](https://www.anaconda.com/products/distribution) distribution platform for Python, we can create a dedicated environment `mycomac` (we can also use a more creative name) for this course.\n",
+    "\n",
+    "For summer term 2022, we use\n",
+    "\n",
+    "- `conda --version`: 4.12.0\n",
+    "- `conda-build --version`: 3.21.8\n",
+    "- `conda create -n mycomac python=3.9 pip numpy sympy scipy matplotlib notebook ipykernel jupyterlab ipympl pydocstyle pycodestyle autopep8 flake8`\n",
```

Comment:
I think it would be much better to provide a `environment.yml` instead.

Reply:
Yeah, that's reasonable, I provided `environment.yml` in the root directory.
Also added `.binder` folder with mybinder config information (thanks for this hint above!)